src/components/UI/Button/Button.tsx

```diff
@@ -64,7 +64,7 @@ export const baseStyleButton = ({ bgColor, textColor }: BaseStyleButtonProps): s
     letter-spacing: 0.01rem;
     min-height: 40px;
     cursor: pointer;
-    border-radius: ${vars.buttonRadius};
+    border-radius: 4px;
```

Comment:
same thing change to use twin marco so can use tailwind class `.rounded`

Reply:
will do in the next pr